src/renderer/models/ssz/types.ts

```diff
@@ -1,9 +1,12 @@
-import {SimpleContainerType} from "@chainsafe/ssz";
+import {SimpleContainerType, Type} from "@chainsafe/ssz";
 
-export const Account = (): SimpleContainerType => ({
+// @ts-ignore
+export const Account: SimpleContainerType = {
+    // @ts-ignore
+    type: Type.container,
```

Comment:
you don't need type though, think AnyContainerType should suffice

Reply:
We need it for the JSON serializer as we want to extract only specific attributes from the object and we need to know if the object passed to the serializer is actually a container or a type literal.
Check the `serializers/json.ts` to see this in action.

But yeah with the AnyContainerType we don't need the ts-ignore.

Reply:
I think in ssz, container is always if it has key "fields"

Reply:
I managed to remove the ts-ignore but I'm still using `type: Type.container` (I switched to `AnyContainerType` as you suggested). I can remove it and instead check if `field` exist and do the container check that way, but the current implementation also works so it is down to personal preference, what do you think? @mpetrunic

Reply:
I would suggest checking if fields prop exists just to be compliant with how ssz is implemented

Reply:
btw you have some lint errors

Reply:
Lint errors are frontend related and I didn't want to touch things that I'm not assigned to.
Errors appeared as a result of the `@typescript-eslint/explicit-member-accessibility`. 
I can fix it in this PR if you are ok with me touching frontend code?

Reply:
you have to or we cannot merge :)

Reply:
oh, ok 😅